chore: update to 0.10.0

xoascf opened the pull request:
Required to upgrade [fastfetch](https://github.com/deepin-community/fastfetch) to [2.23.0](https://github.com/fastfetch-cli/fastfetch/releases/tag/2.23.0).

xoascf committed:
chore: update to 0.10.0

UTsweetyfish commented on debian/changelog:
```diff
@@ -1,3 +1,21 @@
+yyjson (0.10.0-0deepin) unstable; urgency=medium
```
```suggestion
yyjson (0.10.0-0deepin1) unstable; urgency=medium
```

UTsweetyfish reviewed (changes requested):
Please bump VER2 to 1.

xoascf committed:
Update debian/changelog

Co-authored-by: Tianyu Chen <124018391+UTsweetyfish@users.noreply.github.com>

xoascf committed:
Use Debian naming with matching sonames

xoascf committed:
Fix lintian

extended-description-is-empty
latest-changelog-entry-without-new-date
debian-changelog-line-too-long

xoascf committed:
Update control

xoascf committed:
Remove libyyjson-docs.substvars

UTsweetyfish commented:
/topic fastfetch-2.23.0
